backend/app/controllers/api/v1/accounts/users_controller.rb

```diff
@@ -29,6 +29,15 @@ def transfer_ownership
           render json: UserSerializer.new(user, params: {current_user: user}).serializable_hash
         end
 
+        # DELETE
+        def favourites
+          current_user.favourite_projects.destroy_all
```

Comment:
Not better in transaction? Just in `ActiveRecord::Base.transaction`

Reply:
Hmm, yes, it is, I think nothing can blow up here, but just to be on save side ;)